src/components/organisms/TCRecommendedTrendsList/TCRecommendedTrendsList.tsx

```diff
@@ -0,0 +1,28 @@
+import './TCRecommendedTrendsList.scss'
+import { FunctionComponent } from 'react'
+import { TCRecommendedTrend } from '../../molecules/TCRecommendedTrend/TCRecommendedTrend'
+import { IoSettingsOutline } from 'react-icons/io5'
+import { Trend } from '../../../model/Trend'
+
+interface TCRecommendedTrendListProps {
+  trends: Trend[]
+}
+
+export const TCRecommendedTrendList: FunctionComponent<
+  TCRecommendedTrendListProps
+> = ({ trends }) => {
+  return (
+    <div className='trends'>
+      <div className='header'>
+        <p className='header__title'> Trends for you </p>
+        <IoSettingsOutline className='header__settings-icon' />
+      </div>
+      {trends.map((trend) => (
```

Comment:
It is good idea to wrap this into div with class name tc-recommended-trends-list__trends